# Message defining covariance estimator parameters

# The source this covariance estimator corresponds to
string source_name

float64[] parameters